source/jormungandr/tests/kirin_realtime_tests.py

```diff
@@ -1557,6 +1558,20 @@ def test_add_new_trip(self):
         assert status == 404
         assert response['error']['message'] == 'ptref : Filters: Unable to find object'
 
+        # Check that no departure exist on stop_point stop_point:stopC
+        departure_query = "stop_points/stop_point:stopC/departures?_current_datetime=20120614T080000"
+        departures = self.query_region(departure_query)
+        assert len(departures['departures']) == 0
+
+        # Check that no stop_schedule exist on line:additional_service and stop_point stop_point:stopC
+        ss_query = (
+            "stop_points/stop_point:stopC/lines/line:additional_service/"
+            "stop_schedules?_current_datetime=20120614T080000&data_freshness=realtime"
```

Comment:
I vote to also add a check using `base_schedule` to check that we don't contaminate it (departures and/or stop_schedules).

Reply:
This is not in the scope of the _JIRA ticket_, so normally we should leave as is, and create an other PR if it necessary, according to the rules that we have mentionned in Meeting (_Talk to the team -> create new ticket if necessary -> create new PR_).
But I think, it's an oblivion. We should test this _entry point_ in the ticket. So, I am agree with you @pbougue.

Reply:
I added some more verifications as asked